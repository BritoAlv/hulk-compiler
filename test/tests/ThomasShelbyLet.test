let a = "Thomas" in 
{
    let a = a @@ "Shelby" in print(a);
    print(a);
};