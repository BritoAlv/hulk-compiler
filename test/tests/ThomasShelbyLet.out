Exception occurred at PC=0x0040008c
  Bad address in data/stack read: 0x00000000
  Exception 7  [Bad data address]  occurred and ignored
Exception occurred at PC=0x004000a0
  Bad address in data/stack read: 0x00000004
  Exception 7  [Bad data address]  occurred and ignored
Exception occurred at PC=0x004000a4
  Bad address in data/stack read: 0x00000004
  Exception 7  [Bad data address]  occurred and ignored
0 Shelby
Thomas

